doc/data/glossary/glossary.yml

```diff
@@ -0,0 +1,182 @@
+
+Media Access Control:
+  description: LoRaWAN media access protocols, downloadable from the LoRa Alliance. The MAC version for your device should be specified in the product documentation as LoRaWAN specification or protocol version.
+  link: https://lora-alliance.org/lorawan-for-developers
+  abbr: MAC
+
+Regional Parameters:
+  description: LoRaWAN official regional specifications, downloadable from the LoRa Alliance.
+  link: https://lora-alliance.org/lorawan-for-developers
+  abbr: PHY
+
+Physical Layer:
+  description: Hardware pertaining to LoRaWAN official regional specifications. The PHY version for your device should be specified in the product documentation as Regional Parameter version.
+  abbr: PHY
+
+Frequency Plan:
+  description: Frequencies used in geographical areas, based on what is specified in Regional Parameters.
+  link: https://lora-alliance.org/lorawan-for-developers
+
+Local area network:
+  description: A network that connects computers within a limited area.
+  link: https://en.wikipedia.org/wiki/Local_area_network
+  abbr: LAN
+
+Wide area network:
+  description: A network that connects within a wide geographical range, usually to the internet.
+  link: https://en.wikipedia.org/wiki/Wide_area_network
+  abbr: WAN
+
+ISM Radio Bands:
+  description: The Industrial, Scientific, and Medical frequency bands, most commonly used for low-power and short range telecommunications, such as WiFi, Bluetooth, Zigbee, wireless telephones, RFID, and NFC
+  link: https://en.wikipedia.org/wiki/ISM_band
+
+Payload:
+  description: A LoRaWAN packet. Application payloads are decrypted data relevant to the application. Gateway payloads are encrypted and include the application payload plus a frame counter and other LoRaWAN data.
+
+Interval:
+  description: The duration between messages.
+  link: https://www.thethingsnetwork.org/docs/lorawan/duty-cycle.html
+
+Duty Cycle:
+  description: The fraction of time a device is busy. When a single device transmits on a channel for 2 time units every 10 time units, this device has a duty cycle of 20%.
+  link: https://www.thethingsnetwork.org/docs/lorawan/duty-cycle.html
+
+Data Rate:
+  description: A combination of Bandwidth and Spreading Factor which defines how quickly a message is transmitted.
+  link: https://www.thethingsnetwork.org/docs/lorawan/modulation-data-rate.html
+
+Bandwidth:
+  description: LoRaWAN can use channels with a bandwidth of either 125 kHz, 250 kHz or 500 kHz, depending on the region or the frequency plan. Making the bandwidth 2x wider (from BW125 to BW250) allows you to send 2x more bytes in the same time.
+  link: https://www.thethingsnetwork.org/docs/lorawan/modulation-data-rate.html
+
+Spreading Factor:
+  description: Making the spreading factor 1 step lower (from SF10 to SF9) allows you to send 2x more bytes in the same time. Lowering the spreading factor makes it more difficult for the gateway to receive a transmission, as it will be more sensitive to noise.
```

Comment:
```suggestion
  description: The transmission speed or Data Rate of a LoRaWAN message, ranging from SF7 (highest Data Rate) to SF12 (lowest Data Rate). Making the spreading factor 1 step lower (from SF10 to SF9) allows you to roughly send the same amount of data use half the time on air. Lowering the spreading factor makes it more difficult for the gateway to receive a transmission, as it will be more sensitive to noise.
```